### PR TITLE
Blog Post: 5th Wave of Bitcoin Grants

### PR DESCRIPTION
**Pull request by dergigi**
To be reviewed by @OpenSats/volunteers 

**Build preview:**

- [/blog/bitcoin-grants-july-2024](https://os-website-git-5th-wave-bitcoin-open-sats.vercel.app/blog/bitcoin-grants-july-2024)

**Social preview:**

36-bitcoin-jul

### CODE REVIEW

File: data/blog/bitcoin-grants-july-2024.mdx [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+supported devices, including PCs and Android mobile devices. New features will
+include the introduction of Krux Apps to enable games, additional OpSec
+features, and the ability to use the signing device for Nostr.
+
+Repository: [selfcustody/krux](https://github.com/selfcustody/krux)  
+License: Apache2 & [MIT](https://github.com/selfcustody/krux/blob/main/LICENSE.md)
+
+### Krux Installer
+
+Whilst Krux is an open-source firmware that allows anyone to build their own
```
Comment (Arvin21M) [inline data/blog/bitcoin-grants-july-2024.mdx:140]:
```suggestion
Whilst Krux is open-source firmware that allows anyone to build their own
```

**Review by Arvin21M — APPROVED** [review]

**Review by Arvin21M — APPROVED** [review]

**Comment by nvk** [comment]
ACK

**Review by Arvin21M — APPROVED** [review]

**Review by btcschellingpt — APPROVED** [review]
ACK

File: data/blog/bitcoin-grants-july-2024.mdx [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+opens the door to browser-based and smaller low-power devices to become Bitcoin
+nodes—including mobile devices!
+
+This grant supports the activities to complete the P2P tx messaging
+implementation, write up the four BIPs required for `utreexo`, and commence
+building the bridge node implementation within Bitcoin Core. Subsequent work on
+optimizing performance for IBD, increasing efficiency in the accumulator
+implementation, and improving library support are to follow.
+
+Repository: [https://github.com/utreexo/utreexo](https://github.com/utreexo/utreexo)
```
Comment (dergigi) [inline data/blog/bitcoin-grants-july-2024.mdx:233]:
```suggestion
Repository: [utreexo/utreexo](https://github.com/utreexo/utreexo)  
```

**Pull request merged**